MIDAS/platformio.ini

```diff
@@ -6,7 +6,7 @@ extra_scripts =
     pre:extra_script.py
 build_flags =
     -DARDUINO_USB_CDC_ON_BOOT=1
-    -std=gnu++17
+    -std=gnu++2a
```

Comment:
Do newer versions of clang and gcc still support the gnu++2a thing or do you have to say gnu++20?

Reply:
u have to say gnu++2a, gnu++20 didn't work